### PR TITLE
sec docs: explicitly note what happens OUTSIDE the attack model

### PR DESCRIPTION
**Pull request by enkore**
I thought I included this from the start, but obviously I didn't.

### CODE REVIEW

File: docs/internals/security.rst [hunk 1]
```diff
@@ -37,6 +37,9 @@ Under these circumstances Borg guarantees that the attacker cannot
 The attacker can always impose a denial of service per definition (he could
 forbid connections to the repository, or delete it entirely).
 
+In an attack model where multiple clients independently access the same repository,
+Borg fails to provide confidentiality (i.e. guarantees 3) and 4) are essentially void).
```
Comment (ThomasWaldmann) [inline docs/internals/security.rst]: In an extended attack model (same as above, but with multiple clients independently updating the same repository), ...

File: docs/internals/security.rst [hunk 2]
```diff
@@ -37,6 +37,10 @@ Under these circumstances Borg guarantees that the attacker cannot
 The attacker can always impose a denial of service per definition (he could
 forbid connections to the repository, or delete it entirely).
 
+When the above attack model is extended to include multiple clients
+independently updating the same repository, then Borg fails to provide
+confidentiality (i.e. guarantees 3) and 4) do not apply any more).
+
```
Comment (ThomasWaldmann) [inline docs/internals/security.rst]: Hmm, maybe we should add a note to the normal user docs also. Who reads internals docs anyway? :)

Comment (ThomasWaldmann) [reply docs/internals/security.rst]: @enkore did you see my last comment?

Comment (enkore) [reply docs/internals/security.rst]: I didn't, but I don't feel like puzzling over were to put that in the user docs, either.

**Pull request merged**